eng/regenerate-download-table.proj

```diff
@@ -24,6 +24,7 @@
       <Platform Include="Linux (x64)" Parenthetical=" (for glibc based OS)" Abbr="linux-x64" />
       <Platform Include="Linux (armhf)" Parenthetical=" (for glibc based OS)" Abbr="linux-arm"/>
       <Platform Include="Linux (arm64)" Parenthetical=" (for glibc based OS)" Abbr="linux-arm64" />
+      <Platform Include="Linux (loongarch64)" Parenthetical=" (for glibc based OS)" Abbr="linux-loongarch64" />
```

Comment:
This change should be reverted. We are not going to have nightly builds for loongarch64 in foreseeable future.

Reply:
Thanks for your review.
I will revert it.